frontend/public/locales/en/learn/from-work-to-retirement.json

```diff
@@ -0,0 +1,65 @@
+            "retire-from-work": "Retiring from work and claiming your pensions are two different things. You are free to combine your income from work and savings with pensions. However, the later you claim your OAS and CPP, the higher your monthly pensions will be.",
+            "afford-to-delay": "If you can afford to delay your OAS and CPP until age 70, you will increase your OAS monthly pension by more than one-third, and you will more than double your CPP retirement pension, compared to the earliest age when you can claim them.",
+            "public-pensions": "The public pensions provide stable, inflation-proof income that will support you for the rest of your life, and would reduce uncertainty of your private investments and risk of outliving your savings.",
+            "at-age": "If at age 65 you have low income and qualify for the OAS, do not delay as you might receive additional income from the Guaranteed Income Supplement (GIS). You need to be in receipt of OAS to get your GIS, as you will be at a loss, if you delay your OAS. You may also consider starting your CPP earlier than age 65."
+        },
+        "retirement-today": "Retirement today is not what it was ten years ago, as people now have more choices and flexibility to tailor their retirement plans to fit their lifestyle preferences. This new chapter of life could include:",
+        "life-chapters": {
+            "phase-retirement": {
+                "content": "<a>Phased retirement</a>: a gradual transition from full-time to part-time employment and then full retirement.",
+                "link": "https://google.com"
+            },
+            "second-career": {
+                "content": "<a>A second career</a>:  when retirees change their jobs altogether, or open a new business, pursuing their long-time dreams.",
+                "link": "https://google.com"
+            },
+            "full-retirement": {
+                "content": "<a>Full retirement</a> when people retire from work for good.",
+                "link": "https://google.com"
+            },
+            "content": "Learn how you can structure your finances in these cases."
+        }
+    },
+    "phased-retirement": {
+        "heading": "Phased retirement",
+        "sub-heading": "Transition gradually from work to retirement",
+        "content": "Many people are not ready to go from full-time work to full-time retirement immediately. Instead, they prefer to lower their hours of work and ease into retirement, while maintaining some of their original earnings. You can talk to your employer about gradually reducing your hours or working fewer days per week before you retire fully, but this will need to be an agreement between you and your employer. If you decide to pursue a phased retirement, it’s important to understand how it will affect your finances."
```

Comment:
```suggestion
        "content": "Many people are not ready to go from full-time work to full-time retirement immediately. Instead, they prefer to lower their hours of work and ease into retirement, while maintaining some of their original earnings. You can talk to your employer about gradually reducing your hours or working fewer days per week before you retire fully, but this will need to be an agreement between you and your employer. If you decide to pursue a phased retirement, it's important to understand how it will affect your finances."
```